### ISSUE TITLE
Path /etc/elasticsearch/config/certs/http_ca.crt is wrong when trying to verify Elasticsearch is running

### ISSUE THREAD

**Issue by liu-xiao-guo** [elastic/elasticsearch#85824]
https://www.elastic.co/guide/en/elasticsearch/reference/current/rpm.html#rpm-running-systemd

![image](https://user-images.githubusercontent.com/8455957/162904102-ba963af0-93f7-421e-a242-96ccc3584074.png)
<img width="909" alt="image" src="https://user-images.githubusercontent.com/8455957/162904178-7e2ec692-52eb-4c5a-bd62-42b69b888193.png">

**Comment by lockewritesdocs** [comment]
Can someone from @elastic/es-delivery confirm the directory structure on RPM? In the section for [Configuring Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/rpm.html#rpm-configuring) we indicate that:

>The /etc/elasticsearch directory contains the default runtime configuration for Elasticsearch.

It seems that there's no `/config` directory on RPM installations, which differs from MacOS and some others.

**Comment by mark-vieira** [comment]
We have an [open pull request](https://github.com/elastic/elasticsearch/pull/85728) to fix this. Just waiting on the author to sign the CLA. Alternatively we can't just quickly fix this and close that PR.

**Comment by lockewritesdocs** [comment]
Thanks @mark-vieira! I left a comment in #85728. I'm going to fix this issue in a separate PR.
